packages/frontend/src/pages/GroupList.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { FullWidthText } from "../components/Input/Text";
+import { CopyTextButton, SmallButton } from "../components/Input/Buttons";
+import { BigText } from "../components/Input/Text";
+
+export default function NameForm() {
+	return (
+		<div className="flex flex-col space-y-normal justify-center w-5/6">
+			<FullWidthText>Coop Name:</FullWidthText>
+			<CopyTextButton
+				linkText="http://localhost:3000/flock/flyingtaco/join/"
```

Comment:
Can you parametrize the link? Or is that outside the scope of this issue?

Reply:
The link is just a placeholder for now. Parametrizing the link is outside the scope of this issue because it involves backend logic -- this issue is for UI implementation